### DISCUSSION TITLE
How to reduce the number of JS files under assets

### DISCUSSION THREAD

**Discussion by qiaoshouzi** [vuepress/core#571] [Q&A] [answered]
How to reduce the number of JS files under assets

After one of my projects was built, 2000+ JS files are generated

**Comment by Mister-Hope** [comment, marked as answer]
Currenly no, you can choose webpack as default bundler, as each page will generate 2 js files in vite, but 1 in webpack. This is for scalability concern, as you should only load a page's js chunk once you choose to navigate to it.

And @meteorlxy  here, some optimize could be made for vite to make sure it only generate 1 js chunk for 1 page.
